puppet/services/opendaylight-api.yaml

```diff
@@ -32,6 +36,14 @@ parameters:
   OpenDaylightApiVirtualIP:
     type: string
     default: ''
+  OpendaylightPortBindingController:
+    description: Opendaylight port binding controller
+    type: string
+    default: 'pseudo-agentdb-binding'
+  OpendaylightHostconfURI:
```

Comment:
Will change to OpenDaylight. I think we should keep HostconfURI to keep make consistent with networking_odl's option name, odl_hostconf_uri